Gemfile

```
@@ -3,7 +3,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.1.2"
+ruby "3.2.0"
```

quand je vois ça, j'ai toujours envie de remplacer ça par :
```suggestion
ruby File.read(".ruby-version").chomp
```
Mais j'ai du mal à savoir si c'est une idée passable ou catastrophique 🙈

---

Oui, j'avais ça sur certains projets, mais dans la mesure où de toute façon la valeur est répétée dans plein de fichiers, on est pas à un fichier près.

Ou plus concrètement, je pense que c'est une une solution à un non-problème, car je n'ai jamais eu le coup où j'oublie de remplacer une des deux valeurs. :smile_cat: